src/tests/types.ts

```diff
@@ -360,6 +360,10 @@ export type ApexTestResultData = {
    * The start time of the test method.
    */
   testTimestamp: string;
+  /**
+   * Indicates if the results are for a test setup method. The default is false.
+   */
+  isTestSetup?: boolean;
```

Comment:
@CristiCanizales This should appear in at new type `ApexTestResultDataRaw` per spec